docs/book/src/images/cluster-topology-controller.plantuml

```diff
@@ -0,0 +1,33 @@
+@startuml
+
+start
```

Comment:
I like a lot how the diagram abstracts away from the implementation details.

One question though, should we move l.10 down to l.17 (and potentially merge them)? The reconciler first reconciles InfraCluster and ControlPlane and then updates both references in the Cluster at the same time.

Reply:
Good tip.  Updated accordingly.